tests/Doctrine/Tests/Common/Proxy/ProxyLogicIdentifierGetterTest.php

```diff
@@ -78,6 +78,13 @@ public function methodsForWhichLazyLoadingShouldBeDisabled()
             ]
         );
 
+        $methods = array_merge(
+            $methods,
+            [
+                [new LazyLoadableObjectWithTraitClassMetadata(), 'identifierFieldInTrait', 123],
```

Comment:
Why isn't this just in the `array_merge()` block below?

Reply:
Basically I followed the same style since there was already a separate block of methods merged in. I can change them all to just be one big array.

Reply:
Current style seems to be a relic from c0c18190c8e3220e70e279f2c927465a834fa706

Reply:
Since we don't support PHP <7.1 anymore, this can be simplified now :+1:

Reply:
Fixed